shared/util.go

```diff
@@ -1349,7 +1349,7 @@ func RenderTemplate(template string, ctx pongo2.Context) (string, error) {
 
 	// Looks like we're nesting templates so run pongo again
 	if strings.Contains(ret, "{{") || strings.Contains(ret, "{%") {
-		return RenderTemplate(ret, ctx)
+		return "", errors.New("Template contains nested template tags, which is not allowed")
```

Comment:
Shall we limit nesting depth instead to avoid breaking changes? Does the original commit that added that support hint as to a use case we can use as an indicator of a realistic nesting depth limit?

Reply:
This was introduced by #5277 which addressed #2610. I don't see any justifications for recursion but will add it back with a limit of 5 unless you can come up with a better magic number.